src/main/java/manufacturing/model/Car.java

```diff
@@ -0,0 +1,69 @@
+package manufacturing.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
```

Comment:
checklist last point

Reply:
sorry. didn't see checklist in the bottom. removed